### PR TITLE
Feat:Added Rating and Text utilities component

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat:Added Rating and Text utilities component
1.Added Rating component
2.Added Text Utilities component

### CODE REVIEW

**Review by SaurabhChawda — APPROVED** [review]

File: components/Rating.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@import url("/Navigation.css");
+@import url("/document.css");
+.Rating_container{
+    width: 66rem;
+    height: 34.5rem;
```
Comment (t007rushi) [inline components/Rating.css]: the code is breaking for this can use % for right-side container

File: components/Rating.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    margin-top: -69.2rem;
+    margin-left: 15.1rem;
+    overflow-y: scroll;
+    overflow-x: hidden; 
+}
+.Rating{
+    width: 57rem;
+    height:60rem;
```
Comment (t007rushi) [inline components/Rating.css]: same as above

File: components/Rating.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+.rating-paragraph{
+    font-size: 1.3rem;
+}
+.fas{
+    color: #ffb400;
+}
+button{
+    padding: 0.5rem 1rem;
+    margin-top: 1rem;
+    background-color: blue;
```
Comment (t007rushi) [inline components/Rating.css:38]: same as above

File: components/Rating.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    height: 12rem;
+    width: 35rem;
+    margin-left: 2rem;
+    padding-top: 1rem;
+}
+.rating-paragraph{
+    font-size: 1.3rem;
+}
+.fas{
+    color: #ffb400;
```
Comment (t007rushi) [inline components/Rating.css:33]: use color variables

**Review by t007rushi — CHANGES_REQUESTED** [review]

**Review by krishnakant01 — APPROVED** [review]
implement the changes suggested by others and you are good to merge.

**Pull request merged**